src/components/items/selected_item.js

```diff
@@ -5,15 +5,13 @@ import IconButton from "@material-ui/core/IconButton";
 import ItemLabel from "./item_label";
 import classnames from "classnames";
 import styles from "./selected_item.scss";
-import itemStyles from "./item.scss";
 
 const SelectedItem = ({ item, height, group, isLocked }) => (
   <div
     className={classnames({
       [styles.with_grouping]: group,
       [styles.selected_item]: !group,
-      [itemStyles.disabled]: isLocked(item),
-      [itemStyles.selected]: isLocked(item)
+      [styles.disabled]: isLocked(item)
```

Comment:
please check if SelectedItem is something that can be overriden by user. 
If yes - then we should do the job for him, at least calculate is item disabled on higher level and pass result to this renderer. Do not put this responsibility on user.